training/yolo_det_train.py

```diff
@@ -0,0 +1,44 @@
+""" Trains a YOLOv8 model with default parameters.
+
+MODEL can be set to yolov8s.pt, yolov8m.pt, yolov8l.pt, or yolov8x.pt.
+
+DATA is the name of your yaml file containing train, val, and test
+image paths as well as class information.
```

Comment:
Do we have an example YAML file for this in the repo?  If not, I think it might make sense to add one as part of this branch/PR.

Reply:
I added the example YAML file. I named it consistent with the "DATA" parameter in the script.